### ISSUE TITLE
SCK only enriches splunk.com/* prefixed annotations

### ISSUE THREAD

**Issue by schose** [splunk/splunk-connect-for-kubernetes#553]
**What happened**:

I'm adding custom annotations to a pod and try to enrich them as custom fields in logging. It looks like only splunk.com/* prefixed fields are enriched. 

**What you expected to happen**:

I need to read annotations with custom prefixed. 

**How to reproduce it (as minimally and precisely as possible)**:

- create a pod
```
apiVersion: v1
kind: Pod
metadata:
  annotations:
    splunk.com/custom_field: "value custom field"    
    splunk.com/custom_field1: "value custom field1"
    ohmygod.de/custom_field2: "valuecustomfield2"
  name: apachelogs
  namespace: default
spec:
  containers:
  - image: edsiper/apache_logs:latest
    name: apachelogs
```

create helm config:
```
splunk-kubernetes-logging:
  enabled: true
  loglevel: debug
  splunk:
      hec:
        token: d8d17157-a4d3-4bb2-98cd-e636c5e88888
        #host: hfcip.splunk.svc.cluster.local
        indexName: kubernetes
  customMetadataAnnotations:
    - name: custom_field
      annotation: splunk.com/custom_field
    - name: custom_field1
      annotation: splunk.com/custom_field1
    - name: custom_field2
      annotation: ohmygod.de/custom_field2
```

only custom_field and custom_field1 are enriched. 

**Anything else we need to know?**:

configmap does not look that bad.. but i do not see the issue.. 

```
<record>
...
          custom_field1 ${record.dig("kubernetes","annotations","splunk.com/custom_field1") ? record.dig("kubernetes","annotations","splunk.com/custom_field1") : record.dig("kubernetes","namespace_annotations","splunk.com/custom_field1")}
          custom_field2 ${record.dig("kubernetes","annotations","ohmygod.de/custom_field2") ? record.dig("kubernetes","annotations","ohmygod.de/custom_field2") : record.dig("kubernetes","namespace_annotations","ohmygod.de/custom_field2")}
</record>
...
        <fields>
..
          custom_field
          custom_field1
          custom_field2
..
        </fields>

```
**Environment**:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T23:35:15Z", GoVersion:"go1.14.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:51:04Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```
- Splunk version: 8.1.1
- Splunk Connect for Kubernetes helm chart version: 1.4.5

**Comment by rockb1017** [comment]
Thank you for reporting this. this seem to be a quick fix. i will work on this soon

**Comment by rockb1017** [comment]
this is released in 1.4.6
